coalib/settings/SectionFilling.py

```diff
@@ -3,7 +3,6 @@
 from coalib.bears.BEAR_KIND import BEAR_KIND
 from coalib.collecting import Dependencies
 from coalib.collecting.Collectors import collect_bears
-from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
```

Comment:
Don't we run PyUnusedCodeBear - https://github.com/coala/coala/blob/master/.coafile#L15 ?? 
How is this not being detected ?